### PR TITLE
fix Options->General menu pages 1 and 2

### LINKED ISSUE #1151: Clean up Options -> General menu
Oops, the general menu is still a mess with "advanced sound" on its own page. I'd like to have all music and sound related options in the second page and restructure the first page a bit so that it takes a bit more space.

### PR DESCRIPTION
**Pull request by fabiangreffrath**
Fixes #1151

### CODE REVIEW

**Comment by rfomin** [comment]
How about moving some of the "Display options" to the first page, e.g. translucency, level brightness?

Do we need "Doppler effect" and "Air absorption" options? I think we should make them config only.

**Comment by ceski-1** [comment]
> Do we need "Doppler effect" and "Air absorption" options? I think we should make them config only.

Config only should be fine.

**Comment by fabiangreffrath** [comment]
> How about moving some of the "Display options" to the first page, e.g. translucency, level brightness?

Good idea! This would help to fill up the first page a bit. 

> Do we need "Doppler effect" and "Air absorption" options? I think we should make them config only.

This, in turn, would lead to even less items in the second page. 😞

**Comment by rfomin** [comment]
> This, in turn, would lead to even less items in the second page.

Maybe we can move some other submenu to the second page, like mouse options.

**Review by rfomin — APPROVED** [review]
Looks good to me. A little empty space on page 1-2, but I think we will come up with more options over time 😄

File: src/m_menu.c [hunk 1]
```diff
@@ -4064,7 +4051,20 @@ setup_menu_t gen_settings1[] = { // General Settings screen1
...
+   M_Y + gen2_sndchan*M_SPC, {"snd_channels"}},
+
+  {"Pitch-Shifted Sounds", S_YESNO, m_null, M_X,
+   M_Y + gen2_pitch*M_SPC, {"pitched_sounds"}},
+
+  // [FG] play sounds in full length
+  {"Disable Sound Cutoffs", S_YESNO, m_null, M_X,
+   M_Y + gen2_fullsnd*M_SPC, {"full_sounds"}},
+
+  {"", S_SKIP, m_null, M_X, M_Y + gen2_gap3*M_SPC},
```
Comment (ceski-1) [inline src/m_menu.c:4067]: Should this use `gen2_gap1` ?

**Review by ceski-1 — APPROVED** [review]

**Pull request merged**